lib/tasks/export.rake

```diff
@@ -0,0 +1,13 @@
+namespace :mongo do
```

Comment:
Might wanna stick a TODO on this as well (assuming it's meant to be cleaned up after the migration)

Reply:
Yeah, when it's clean-up time we'll basically do a global find for 'mongo' and delete pretty much everything. Happy to add a TODO though